d3ploy/tester.py

```diff
@@ -218,44 +218,54 @@ def supply_demand_dict_nond3ploy(sqlite, commod, demand_eq=0):
     return all_dict
 
 
-def residuals(all_dict):
-    """ Conducts a chi2 goodness of fit test
+def residuals_under(all_dict):
+    """ obtains the cumulative under supply
     Parameters
     ----------
     dict_demand: timeseries dictionary of demand values
     dict_supply: timeseries dictionary of supply values
     Returns
     -------
-    returns an int of the chi2 (goodness of fit value) for
-    the two input timeseries dictionaries
+    returns a double as the cumulative difference between demand
```

Comment:
I strongly recommend numpy style docstrings... 


```suggestion
    ss_res : float
        The cumulative difference between demand
```

Reply:
ok